### DISCUSSION TITLE
Wire:model help

### DISCUSSION THREAD

**Discussion by codemonkey76** [livewire/livewire#3528] [1. Help] [answered]
I have a blade component that is using alipinejs to $dispatch('input', 'some-data') 
i am using wire:model on this component but there is no ajax request that happens when i trigger that event, am I missing something?

**Comment by ryatkins** [comment]
@codemonkey76 do you have a listener somewhere that is making an action happen to call Livewire (trigger an ajax event)?

If you're just trying to call a method on your Livewire component you can use $wire
```
<input type="text" wire:model.defer="myData" />
<button @click="$wire.save">Save</button>
```

```
public function save()
{
    // do something with $this->myData
}

```

**Comment by joshhanley** [comment, marked as answer]
I just saw this. @ryatkins we managed to solve it on discord. 

The issue was that `{{ $attributes }}` weren't being output anywhere, so no `wire:model` was ending up in the rendered DOM, hence no listener was being setup to listen for the input event.

**Reply by ryatkins** [reply]
Took my best guess at this one LOL.
